packages/help-center/src/components/utils.tsx

```diff
@@ -3,6 +3,24 @@ import Smooch from 'smooch';
 import type { ContactOption } from '../types';
 import type { ZendeskConversation, SupportInteraction } from '@automattic/odie-client';
 
+const isMatchingInteraction = (
+	supportInteraction: SupportInteraction,
+	supportInteractionId: string
+): boolean => {
+	return supportInteraction.uuid === supportInteractionId;
+};
+
+const filterConversations = (
```

Comment:
Nit: name could be clearer. What kind of filter are we applying? `filterConversationsBySupportInteractions` or something would make it quite clear.